frontend/src/components/GemMap.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <h4>Hello Friendos</h4>
+    Latitude: {{ currPosition.lat.toFixed(2) }}, Longitude:
+    {{ currPosition.lng.toFixed(2) }}
+  </div>
+
+  <google-map-loader
+    :mapConfig="mapConfig"
+    :apiKey="apiKey"
+    :markersOptions="markersOptions"
+  />
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue";
+import { useGeolocation } from "../useGeolocation";
+import GoogleMapLoader from "./GoogleMapLoader.vue";
+
+const DEFAULT_MAP_CONFIG = {
```

Comment:
open qn: when the user first opens the website, should they see the overview of sg (current), or a more zoomed in view centered on their own location (like gmaps)

Reply:
i think it should be the latter! i've made the changes